manifests/f/FlorianHoech/DisplayCAL/3.8.9.3/FlorianHoech.DisplayCAL.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: FlorianHoech.DisplayCAL
+PackageVersion: 3.8.9.3
+PackageLocale: en-US
+Publisher: Florian Hoech

```

Comment:
```suggestion
Publisher: Florian Höch
```

Reply:
This package was previously removed because of the special characters. https://github.com/microsoft/winget-pkgs/pull/20031

Reply:
This package was previously removed because of the special characters. https://github.com/microsoft/winget-pkgs/pull/20031

Reply:
It was removed due to pecial characters in the folder name. You can use special characters on everything except folders.